Classes/Service/UsageDetailsService.php

```diff
@@ -326,7 +333,7 @@ protected function usageIsAccessible(string $workspaceName): bool
      */
     protected function buildNodeUri(?Site $site, NodeInterface $node): string
     {
-        $serverRequest = ServerRequest::fromGlobals();
+        $serverRequest = $this->bootstrap->getActiveRequestHandler()->getHttpRequest();
```

Comment:
Hey,
yes, I just read your comment after I created this PR. No problem, I will integrate the check and use `ServerRequest::fromGlobals` as fallback. :) I didn't know that this could fail in the CLI. Thx for your hint.